Assets/Scripts/DialogueBubble.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System.Collections;
```

Comment:
Can you make a new folder for these Dialogues please? E.g. `Assets/Scripts/Dialogue`

Reply:
It's only one script pertaining to dialogue, should I still do it?

Reply:
Either that, or make a `UI` script folder maybe? I'm sure we'll have more scripting elements for UI later on

Reply:
This can be done afterward, it's a small thing!

Reply:
Will make a folder when we identify more scripts that will be used in UI